assets/js/scripts.js

```diff
@@ -194,9 +220,51 @@ if($("#main").hasClass("archive")){
                 });
             } else {
                 $('#loadmore').remove();
-            }	
+            }   
+        },
+
+        loadedScripts: {},
+
+        getScripts: function(scripts){
+            var promise = $.Deferred();
+            var loader = $.proxy(function(){
+                if(scripts.length > 0){
+                    var path = scripts.shift();
+
+                    if(path in this.loadedScripts){
+                        loader();
+                    } else {
+                        $.getScript(this.siteurl + path)
+                        .then($.proxy(function(){
+                            this.loadedScripts[path] = true;
+                            loader();
+                        }, this))
+                        .fail(function(err){
+                            if(err){
+                                promise.rejectWith(err);
+                            } else {
+                                promise.reject();
+                            }
+                        });
+                    }
+                } else {
+                    promise.resolve();
+                }
+            }, this);
+
+            if(this.scriptsPresent){
+                promise.resolve();
+            } else {
+                loader();
+            }
+            
+            return promise;
+        },
+
+        getScript: function(path){
+            return this.getScripts([path]);
+        }
+    };
 
-            return false;
-        });
-    });
-}
\ No newline at end of file
+    $($.proxy(Saga.initialize, Saga));
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Expected an assignment or function call and instead saw an expression.